examples/update-demo/v1beta3/kittern-rc.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1beta3
+kind: ReplicationController
+metadata:
+  name: update-demo-kitten
+spec:
+  replicas: 0
+  selector:
+    name: update-demo
+    version: kitten
+  template:
+    metadata:
+      labels:
+        name: update-demo
+        version: kitten
+    spec:
+      containers:
+      - capabilities: {}
+        image: kubernetes/update-demo:kitten
+        imagePullPolicy: IfNotPresent
```

Comment:
I'm on the fence about default values, in general. On one hand, it unnecessarily makes the example look more complicated. On the other hand, it makes people aware of what's available. In this case, I don't think most people should change imagePullPolicy, so I think we should remove it.